solana-programs/start.sh

```diff
@@ -121,7 +124,7 @@ cat <<EOF
     "signerGroup": "$signer_group",
     "feePayerWallet": $(cat feepayer.json),
     "feePayerWalletPubkey": "$feepayer_pubkey",
-    "ownerWallet": "$owner_wallet",
+    "ownerWallet": $owner_wallet,
```

Comment:
any reason we lost these quotes?

Reply:
was looking to make this equal to feepayerWallet above - wdyt is better?